.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test
+# https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  test-vers:
+    strategy:
+      matrix:
+        node: ['8.6.0', '8', '10', '12', '14']
```

Comment:
why both 8.6 and 8?

Reply:
Copying apm-agent-nodejs in that somewhat: test the latest of each supported release stream, plus the min ver (v8.6.0)

I suppose the travis.yml I'm deleting there included version 6, and it looks like GH Actions still supports a v6 version: https://github.com/actions/node-versions/blob/main/versions-manifest.json#L3278

Reply:
I've dropped the 8.6.0 special -> testing that min ver belongs in apm-agent-nodejs.git.